api/lib/proxy.ts

```diff
@@ -12,38 +12,43 @@ import {
 } from '../configuration/references'
 
 import { http } from './http'
-//import * as log4jui from './log4jui'
 import { EnhancedRequest } from './models'
+import {exists} from './util'
 
-//const logger = log4jui.getLogger('proxy')
-
-export function setHeaders(req: EnhancedRequest) {
+export function setHeaders(req: express.Request) {
```

Comment:
can this be simplified to Request

Reply:
Ideally we need a clean up task of the node layer, including removing of ```EnhancedRequest``` however this PR isn't the place for it :)